util/util.go

```diff
@@ -9,6 +9,10 @@ import (
 	"strings"
 )
 
+const (
+	OSRELEASE_PATH = "/etc/os-release"
```

Comment:
The supervisor components and paths are all resin-specific too, as is the remainder of the tool. We should be careful to make sure the naming isn't confusing, because as-is it implies the provisioner is divided into resin and non-resin.

And if there are shared components we should probably divide them into packages that make sense for whatever the utility is doing, if it's useful enough to be shared then it's not just a helper, it's more of a higher level component, in which case 'util' seems inappropriate.